### PR TITLE
iop:colorin: Correctly check return value

### PR DESCRIPTION
**Pull request by cryptomilk**
Correctly check return value of `dt_imageio_avif_read_color_profile()`

### CODE REVIEW

**Comment by cryptomilk** [comment]
img->profile_size is set if `cp` is filled, the out parameter of dt_imageio_avif_read_color_profile.

Inspect with `git show -U20`

File: src/iop/colorin.c [hunk 1]
```diff
@@ -1924,8 +1924,13 @@ void reload_defaults(dt_iop_module_t *module)
       struct avif_color_profile cp = {
           .type = DT_COLORSPACE_NONE,
       };
+      dt_imageio_retval_t ret;
```
Comment (TurboGit) [inline src/iop/colorin.c:1927]: const dt_imageio_retval_t ret = dt_imageio_avif_read_color_profile(filename, &cp);

File: src/iop/colorin.c [hunk 2]
```diff
@@ -1924,8 +1924,13 @@ void reload_defaults(dt_iop_module_t *module)
...
 
-      img->profile_size = dt_imageio_avif_read_color_profile(filename, &cp);
+      ret = dt_imageio_avif_read_color_profile(filename, &cp);
+      if (ret != DT_IMAGEIO_OK) {
```
Comment (TurboGit) [inline src/iop/colorin.c:1930]: { should be next line.

**Review by TurboGit — CHANGES_REQUESTED** [review]
Minor style fixes.

File: src/iop/colorin.c (hunk 1 above)
Comment (cryptomilk) [reply src/iop/colorin.c]: Fixed.

File: src/iop/colorin.c (hunk 2 above)
Comment (cryptomilk) [reply src/iop/colorin.c]: Fixed.

**Comment by LebedevRI** [comment]
Presumably rawspeed changes weren't meant to be here.

**Comment by TurboGit** [comment]
I'll merge manually removing the rawspeed update.

**Pull request closed without merging**